# costom sensor & method status format.
# 1 for on, 0 for off
Header header

bool lidar
bool imu
bool gps

bool LO
bool LIO
bool LIGO
bool GIO